src/Makefile

```diff
@@ -17,6 +18,54 @@ IOS_BUILD_DIR=$(BUILD_DIR)/ios
 WATCH_BUILD_DIR=$(BUILD_DIR)/watch
 TVOS_BUILD_DIR=$(BUILD_DIR)/tvos
 
+MACOS_DOTNET_BUILD_DIR=$(DOTNET_BUILD_DIR)/macos
+IOS_DOTNET_BUILD_DIR=$(DOTNET_BUILD_DIR)/ios
+WATCHOS_DOTNET_BUILD_DIR=$(DOTNET_BUILD_DIR)/watchos
+TVOS_DOTNET_BUILD_DIR=$(DOTNET_BUILD_DIR)/tvos
+
+GENERATOR_FLAGS=-process-enums -core -nologo -nostdlib -noconfig -native-exception-marshalling --ns:ObjCRuntime
+
+DOTNET_FLAGS=/noconfig /nostdlib+ /deterministic /features:strict /nologo /target:library /debug /unsafe /define:DOTNET /define:DOTNET_TODO \
```

Comment:
Any particular reason to go with `DOTNET` define instead of the standard `NETCOREAPP` one?

Reply:
It seems just `NET` will be the define for the future: https://github.com/terrajobst/designs/blob/net5/accepted/2020/net5/net5.md#preprocessor-symbols, and `NETCOREAPP` is only defined for backwards compatibility. I'll change it to `NET` to be consistent.

Reply:
Fixed to use `NET` (and `NET_TODO`).